src/openstudio_lib/MainRightColumnController.cpp

```diff
@@ -729,15 +729,6 @@ void MainRightColumnController::configureForFacilitySubTab(int subTabID) {
   myLibraryList->setItemsRemoveable(false);
   myLibraryList->setItemsType(OSItemType::LibraryItem);
 
-  myLibraryList->addModelObjectType(IddObjectType::OS_Fan_ZoneExhaust, "Fan Zone Exhaust");
```

Comment:
Why are these removed?

Reply:
https://github.com/openstudiocoalition/OpenStudioApplication/pull/820/commits/d97d3a02bed63ada6b09a124179198cb4496518f

 Commit message: "Unrelated but remove ZoneHVAC entries in Facility and Spaces tab, they make no sense".

You can't assign a ZoneHVAC Component (eg a ZoneHVAC:PackagedTerminalHeatPump) to a space or the building.